server/protocols/jmap-rfc-8621-integration-tests/distributed-jmap-rfc-8621-integration-tests/src/test/resources/jmap.properties

```diff
@@ -1 +1,2 @@
-url.prefix=http://domain.com
\ No newline at end of file
+# Configuration urlPrefix for JMAP routes.
+# url.prefix=http://domain.com
```

Comment:
Why this change?

Reply:
I want to add description, also comment property by accident.I will remove `#`.